Discovery/Discovery/Presentation/NativeDiscovery/DiscoveryViewModel.swift

```diff
@@ -37,7 +37,7 @@ public class DiscoveryViewModel: ObservableObject {
     let connectivity: ConnectivityProtocol
     private let interactor: DiscoveryInteractorProtocol
     private let analytics: DiscoveryAnalytics
-    private let storage: CoreStorage
```

Comment:
Any specific reason to make it public? I thought we can create getter of this property.

Reply:
Thank you for your question. The reason for making `connectivity` public is because it is directly used inside the view. While it is possible to create a public getter and keep the property private, making the property itself public simplifies access in this case, especially since there are no setters and the property is effectively read-only. This ensures that the view has direct and straightforward access to the property without additional boilerplate code.

When we started the project, we decided to follow this approach for consistency. Although your point is valid, it is not critical in this context, and to maintain a uniform coding style, it is better to stick to the initially chosen method.